templates/focus-ios.xliff

```diff
@@ -729,6 +749,74 @@
         <source>URL to open</source>
         <note>Label for input to set a favorite URL to be opened by Siri.</note>
       </trans-unit>
+      <trans-unit id="Tip.Biometric.Title">
+        <source>Lock %@ when a site is open:</source>
+        <note>Text for a label that indicates the title for biometric tip.</note>
```

Comment:
Please call out in comment and explained what it will be replaced with

Reply:
@Delphine We only show this if there are multiple trackers blocked. It will never be 1 tracker always multiple. Will that work?